src/jit/codegenlegacy.cpp

```diff
@@ -19041,7 +19041,10 @@ regMaskTP CodeGen::genCodeForCall(GenTreeCall* call, bool valUsed)
                             indCallReg = regSet.rsGrabReg(RBM_ALLINT); // Grab an available register to use for the CALL
                                                                        // indirection
                             instGen_Set_Reg_To_Imm(EA_HANDLE_CNS_RELOC, indCallReg, (ssize_t)addr);
-                            getEmitter()->emitIns_R_R_I(INS_ldr, EA_PTRSIZE, indCallReg, indCallReg, 0);
+#ifdef _TARGET_ARM_
+                            if (!compiler->opts.ShouldUsePInvokeHelpers())
```

Comment:
@jkotas Sorry I did not catch you quite well. RyuJIT doesn't use ```getAddressOfPInvokeFixup``` and ```getAddressOfPInvokeFixup``` does not have ```CORINFO_CONST_LOOKUP lookup``` argument as ```getAddressOfPInvokeTarget``` does

jit/lower.cpp (RyuJIT)
```
        CORINFO_CONST_LOOKUP lookup;
        comp->info.compCompHnd->getAddressOfPInvokeTarget(methHnd, &lookup);

        void* addr = lookup.addr;
        switch (lookup.accessType)
```

We can fix 
```
private void* getAddressOfPInvokeFixup(CORINFO_METHOD_STRUCT_* method, ref void* ppIndirection)
```
to return 
```
return &pLookup.addr;
```
instead of
```
return pLookup.addr;
```